blockchain/block.go

```diff
@@ -70,3 +70,14 @@ func DecodeBlock(r io.Reader) *Block {
 	gob.NewDecoder(r).Decode(&b)
 	return &b
 }
+
+// ContainsTransaction returns true and the transaction itself if the Block
+// contains the transaction.
```

Comment:
Seems like the right thing to do doesn't it. Breaks my `if block.ContainsTransaction(t) then` idiom 😢